node-operators/block-producers/offboarding/account.md

```diff
@@ -49,7 +49,7 @@ Head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://fraa-dancebox
 
 ### Execute the Pending Request {: #execute-pending-request }
 
-You'll need to wait at least two sessions before executing the pending request. For example, if you submitted the undelegation request at session `1`, the earliest it can be executed is session `3`. Each session is comprised of `{{ networks.dancebox.session.blocks }}` blocks and translates to about {{ networks.dancebox.session.display }} hour per session. So, two sessions correspond to approximately {{ networks.dancebox.staking.staking_session_delay.hours.display }} hours.
+You'll need to wait at least {{ networks.dancebox.staking.leaving_delay_sessions }} sessions before executing the pending request. Each session is comprised of `{{ networks.dancebox.session.blocks }}` blocks and translates to about {{ networks.dancebox.session.hours }} hour per session. So, {{ networks.dancebox.staking.leaving_delay_sessions }} sessions correspond to approximately {{ networks.dancebox.staking.leaving_delay_hours }} hours.
```

Comment:
Ideally these variables would be spelled out like "two sessions" instead of "2 sessions"

Reply:
really? I feel that just create a lot of overhead of maintaining a text version and a number version of the same variable

Will do it tho